src/EditorFeatures/Test/BraceHighlighting/AbstractBraceHighlightingTests.cs

```diff
@@ -19,6 +20,8 @@ public abstract class AbstractBraceHighlightingTests
         {
             using (var workspace = await CreateWorkspaceAsync(markup))
             {
+                WpfTestCase.RequireWpfFact($"{nameof(AbstractBraceHighlightingTests)}.{nameof(TestBraceHighlightingAsync)} creates asynchronous taggers");
```

Comment:
@jasonmalinowski right but seems like you could save yourself some typing if you had two parameters:

- reason for why 
- [CallerMemberInfo] for where (optional)

That would allow this to become:

```
 WpfTestCase.RequireWpfFact("creates asynchronous taggers");
```
Just a suggestion though.